org.eclipse.winery.topologymodeler.ui/src/app/palette/palette.component.ts

```diff
@@ -236,19 +242,23 @@ export class PaletteComponent implements OnInit, OnDestroy {
      * @param name
      * @return result
      */
-    private getNewNodeDataFromNodeTypes(name: string): any {
+    public getNewNodeDataFromNodeTypes(name: string): any {
```

Comment:
Don't you know the type of the return?

Reply:
This object is used only in this method and has no Class/Type right now